### PR TITLE
Use transitionEnd in QUnit since we moved away from PhantomJS

### PR DESCRIPTION
**Pull request by Johann-S**
Remove that code because it's useless since we moved away from PhantomJS, plus by doing that our coverage increased a lot 😍

### CODE REVIEW

File: js/src/util.js [hunk 1]
```diff
@@ -14,7 +14,9 @@ const Util = (($) => {
    * ------------------------------------------------------------------------
    */
 
-  let transition = false
+  const transition = {
```
Comment (XhmikosR) [inline js/src/util.js]: Do we really need it to be an object now?

**Comment by Johann-S** [comment]
Done @XhmikosR, plus I removed `Util.supportsTransitionEnd` which is useless now 👍

**Comment by XhmikosR** [comment]
I wonder, won't this break stuff for people using `Util.supportsTransitionEnd()`?

**Comment by Johann-S** [comment]
it can be a breaking change but if we let this method it will be the same as : 

```js
supportsTransitionEnd() {
  return true
}
```

because we are compatible with browsers whose support transition end.

**Comment by XhmikosR** [comment]
Yeah, I think we should keep compatibility and add a todo comment that this should be removed in v5

**Comment by Johann-S** [comment]
I did that 😉 `Util.supportsTransitionEnd` is back with a TODO

**Comment by XhmikosR** [comment]
Thanks!

How about `transitionEndTest()`? Maybe we should do the same for this too?

**Comment by Johann-S** [comment]
`transitionEndTest` isn't callable currently, so it's not a breaking change

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**